src/tokenize/tokenize.ts

```diff
@@ -53,8 +53,7 @@ function rstrip(input: string, what: string): string {
 }
 
 const Whitespace = "[ \\f\\t]*";
-const Comment_ = "#[^\\r\\n]*";
-const _Ignore = Whitespace + any("\\\\\\r?\\n" + Whitespace) + maybe(Comment_);
```

Comment:
what's going on here? 😄

Reply:
Just totally unused - exist in tokenize.py - but they're also unused there...